src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -0,0 +1,8 @@
+(ns status-im.ui.screens.wallet.send.events
+  (:require [re-frame.core :as re-frame]
+            [status-im.ui.screens.wallet.db :as wallet.db]))
+
+(re-frame/reg-event-db
```

Comment:
imo, we can use `(dispatch [:set-in [:wallet :send :recipient] recipient ])` it's readable right from view, it's just discussion i don't say we need to do it here